getting-started.md

```diff
@@ -0,0 +1,33 @@
+Getting started
+===============
+This section describes how to get started using a PELUX basline build.
+
+The general procedure involves writing the built image to e.g. a USB-stick
+or SD-card, and booting up a target using that. The following sections are divided
+in the steps involved, and have sub-sections where there is a difference
+between the hardware targets.
+
+These instructions assumes there is an image built according to the
+[README.md](README.md), with the following additional assumptions:
+
+* The top level of the build is located in `$HOME/pelux-baseline`
+* In that top level directory there is a directory named `build`
+* The `pelux-intel.xml` manifest was used
+* The `core-image-pelux` image was built
+
+Writing the image
+-----------------
+Poky provides a conveniece script for writing images. Assuming using an unmounted
+USB-stick or SD-card with an ext3 partition, do:
```

Comment:
Is the ext3 partition mandatory in this step? Isn't the whole device repartitioned and overwritten?